Apps/AdminWebClient/src/ClientApp/src/components/covidTreatmentAssessment/CovidTreatmentAssessment.vue

```diff
@@ -0,0 +1,488 @@
+<script lang="ts">
+import { library } from "@fortawesome/fontawesome-svg-core";
+import { faEye, faEyeSlash } from "@fortawesome/free-solid-svg-icons";
+import { DateTime } from "luxon";
+import { extend, ValidationObserver, ValidationProvider } from "vee-validate";
+import { oneOf, regex, required } from "vee-validate/dist/rules";
+import Vue from "vue";
+import { Component, Emit } from "vue-property-decorator";
+
+import BannerFeedbackComponent from "@/components/core/BannerFeedback.vue";
```

Comment:
We'll want to see if we can re-use the banner component that's on the parent view.